server/tests/setupTest.js

```diff
@@ -0,0 +1 @@
+require('babel-polyfill');
```

Comment:
what is this??

Reply:
Helper which I added use async-await. And `Jest` doesn't recognize those statements even if I didn't write any test for it (maybe it analyzes all files which it has to test). So I found some solutions: https://stackoverflow.com/questions/42535270/regeneratorruntime-is-not-defined-when-running-jest-test and with this line `jest` doesn't throw any errors...